dockerfiles/sidecar/Dockerfile

```diff
@@ -19,8 +26,9 @@ ENV SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.
     SUPERCRONIC=supercronic-linux-amd64 \
     SUPERCRONIC_SHA1SUM=5ddf8ea26b56d4a7ff6faecdd8966610d5cb9d85
 
-COPY cron/backup-cron-job /etc/crontabs/backup-cron-job
-COPY scripts scripts
+COPY --from=builder /workspace-data-sync/dockerfiles/sidecar/scripts/watcher ./watcher/watcher.go
+COPY --from=builder /workspace-data-sync/dockerfiles/sidecar/cron/backup-cron-job /etc/crontabs/backup-cron-job
+COPY --from=builder /workspace-data-sync/dockerfiles/sidecar/scripts /scripts
 
 # Add user that will be able to start watcher binary but nothing more
 # the result will be propagated then into scratch image
```

Comment:
this comment makes no sense... we're not running a scratch image.. we're running alpine:3.11... which is 4 versions behind latest 3.15.

Reply:
this should just say "runtime image" so it can be consistent with upstream alpine dockerfile